src/components/Icons/Svg/Massa/Certificate.tsx

```diff
@@ -0,0 +1,53 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React from 'react';
+
+import { ComponentPropsWithoutRef } from 'react';
+
+interface SVGProps extends ComponentPropsWithoutRef<'div'> {
+  size?: number;
+}
+
+export function Certificate(props: SVGProps) {
+  let { size } = props;
+
+  return (
+    <>
+      <svg
+        width={size ?? 16}
+        height={size ?? 16}
+        viewBox="0 0 16 17"
```

Comment:
Double check your export because in the Designs they are squared correctly.

<img width="317" alt="Screenshot 2023-06-12 at 12 49 18" src="https://github.com/massalabs/ui-kit/assets/126461030/ee7232e8-99ea-493b-8e4d-fbafdd9d9e63">

Reply:
![Screenshot from 2023-06-12 14-04-24](https://github.com/massalabs/ui-kit/assets/90157528/a02a36ef-3412-43a6-9136-c85c332e458e)

I reexported and updated the code it is set to 17 by default

Reply:
We wont have this functionally now. Even you add this SVG we won't have for v0.